sample-deploy/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM BASEIMAGE
+COPY adapter /
+ENTRYPOINT ["/adapter"]
```

Comment:
add `--logtostderr` to the entrypoint.  There's basically no condition where you wouldn't want this